### ISSUE TITLE
metrics/test.rmse is NAN for epoch performance evaluation. is that a problem?

### ISSUE THREAD

**Issue by mikeumo** [DeepLabCut/DeepLabCut#2697]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

As I train my network and get output of performance evaluation every 10 epochs, I get the following lines:

metrics/test.rmse:  nan
metrics/test.rmse_pcutoff:nan

### Operating System

macOS Sonoma 14.5

### DeepLabCut version

dlc version 3.0.0rc3

### DeepLabCut mode

single animal

### Device type

gpu (builtin)

### Steps To Reproduce

create a project, pick a training video, extract frames, label frames, create a training dataset, train network...

### Relevant log output

```shell
Epoch 1/200 (lr=0.0001), train loss 0.00063
Epoch 2/200 (lr=0.0001), train loss 0.00011
Epoch 3/200 (lr=0.0001), train loss 0.00011
Epoch 4/200 (lr=0.0001), train loss 0.00005
Epoch 5/200 (lr=0.0001), train loss 0.00004
Epoch 6/200 (lr=0.0001), train loss 0.00003
Epoch 7/200 (lr=0.0001), train loss 0.00005
Epoch 8/200 (lr=0.0001), train loss 0.00006
Epoch 9/200 (lr=0.0001), train loss 0.00003
Training for epoch 10 done, starting evaluation
Epoch 10 performance:
metrics/test.rmse:  nan
metrics/test.rmse_pcutoff:nan
metrics/test.mAP:   0.000
metrics/test.mAR:   0.000
metrics/test.rmse_detections:3.738
metrics/test.rmse_detections_pcutoff:3.738
Epoch 10/200 (lr=0.0001), train loss 0.00002, valid loss 0.00002
Epoch 11/200 (lr=0.0001), train loss 0.00002
Epoch 12/200 (lr=0.0001), train loss 0.00002
Epoch 13/200 (lr=0.0001), train loss 0.00002
Epoch 14/200 (lr=0.0001), train loss 0.00002
Epoch 15/200 (lr=0.0001), train loss 0.00002
Epoch 16/200 (lr=0.0001), train loss 0.00002
Epoch 17/200 (lr=0.0001), train loss 0.00002
Epoch 18/200 (lr=0.0001), train loss 0.00002
Epoch 19/200 (lr=0.0001), train loss 0.00002
Training for epoch 20 done, starting evaluation
Epoch 20 performance:
metrics/test.rmse:  nan
metrics/test.rmse_pcutoff:nan
metrics/test.mAP:   0.000
metrics/test.mAR:   0.000
metrics/test.rmse_detections:3.528
metrics/test.rmse_detections_pcutoff:3.528
Epoch 20/200 (lr=0.0001), train loss 0.00002, valid loss 0.00002
Epoch 21/200 ... etc
```

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
seems there is something off with your data; did you run check labels and verify they look correct?

**Comment by mikeumo** [comment]
Sure did. The labels look exactly where I placed them in the "Check Labels"

**Comment by MMathisLab** [comment]
Is it possible the pycocotools is not installed? Otherwise I imagine the model looks trained (low rmse), so that's my best guess :)

**Comment by n-poulsen** [comment]
@mikeumo If you run inference on your video, do you get any errors? Or does the model make predictions correctly?

**Comment by mikeumo** [comment]
When I train using multiple videos I do not get the error .

**Comment by ingridlreyes** [comment]
@mikeumo from how many vieos doesn´t you get the error? and how many frames? I have the same problem.😞

**Comment by MMathisLab** [comment]
@ingridlreyes can you post an image of the training loss? it might be your learning rate is too high (and we would need more infor to help you, as the code changed a lot since August :D)

**Comment by mikeumo** [comment]
I stopped getting that error :(. I think it disappeared after I increased batch_size or the number of training/testing images, not quite sure. I think I started using at least six videos and several hundred frames. It kind of does not make sense because the input into training is a combined dataset with individual frames and it does not matter where those frames came from.

**Comment by ingridlreyes** [comment]
@MMathisLab 
Hi! Thanks for responding <3
Basically, I've had several issues throughout the training. I have a Mac m3, so I first created the environment in conda with the following file:
name: DEEPLABCUT_M3
channels:
  - defaults
dependencies:
  - python=3.10
  - conda-forge::pip
  - pip:
    - pytorch
    - torchvision
    - torchaudio
  - conda-forge::ipython
  - conda-forge::jupyter
  - conda-forge::nb_conda
  - conda-forge::notebook<7.0.0
  - conda-forge::python.app
  - conda-forge::ffmpeg
  - conda-forge::pytables==3.8.0
  - pip: 
    - 'git+https://github.com/DeepLabCut/DeepLabCut.git@pytorch_dlc#egg=deeplabcut[gui,apple_mchips]'

I only used this environment to generate the frames and labels, then use for video analysis, but now I have this error! 😣 examples/COLAB/COLAB_YOURDATA_TrainNetwork_VideoAnalysis.ipynb
<img width="644" alt="Image" src="https://github.com/user-attachments/assets/f3606407-fae2-43c4-aa69-a9784834e40c" />
